webui/react/src/components/FilterForm/components/FilterField.tsx

```diff
@@ -245,8 +245,8 @@ const FilterField = ({
             )}
             {currentColumn?.type === V1ColumnType.NUMBER && (
               <InputNumber
-                className={css.fullWidth}
                 value={fieldValue != null ? Number(fieldValue) : undefined}
+                width="100%"
```

Comment:
I see that `width` needs to be passed like this to apply to elements deeper down in the Hew component.

Is it accurate that comes from us wanting to show a Label and that moving from Antd built-in to the ConditionalWrapper to display the label or not?